ocs_ci/templates/ocp-deployment/install-config-aws-upi_rhel_workers.yaml.j2

```diff
@@ -0,0 +1,41 @@
+apiVersion: {{ api_version | default('v1') }}
```

Comment:
Is this file somewhere used or just leftover from previous design?

Reply:
@petr-balogh , This I just copied as is from aws-upi.yaml.j2.

Reply:
@petr-balogh , thanks for pointing that. Deployment worked without this file as well. Removed it now.